--- ./src/modules/alsa/alsa-util.c	2021-09-20 18:54:46.297343593 +0000
+++ ./src/modules/alsa/alsa-util.c	2021-09-20 18:34:45.125967255 +0000
@@ -1165,13 +1165,13 @@
 
         PA_ONCE_BEGIN {
             char *dn = pa_alsa_get_driver_name_by_pcm(pcm);
-            pa_log(_("snd_pcm_avail() returned a value that is exceptionally large: %lu bytes (%lu ms).\n"
+            pa_log_debug(_("snd_pcm_avail() returned a value that is exceptionally large: %lu bytes (%lu ms).\n"
                      "Most likely this is a bug in the ALSA driver '%s'. Please report this issue to the ALSA developers."),
                    (unsigned long) k,
                    (unsigned long) (pa_bytes_to_usec(k, ss) / PA_USEC_PER_MSEC),
                    pa_strnull(dn));
             pa_xfree(dn);
-            pa_alsa_dump(PA_LOG_ERROR, pcm);
+            pa_alsa_dump(PA_LOG_DEBUG, pcm);
         } PA_ONCE_END;
 
         /* Mhmm, let's try not to fail completely */
@@ -1228,14 +1228,14 @@
 
         PA_ONCE_BEGIN {
             char *dn = pa_alsa_get_driver_name_by_pcm(pcm);
-            pa_log(_("snd_pcm_delay() returned a value that is exceptionally large: %li bytes (%s%lu ms).\n"
+            pa_log_debug(_("snd_pcm_delay() returned a value that is exceptionally large: %li bytes (%s%lu ms).\n"
                      "Most likely this is a bug in the ALSA driver '%s'. Please report this issue to the ALSA developers."),
                    (signed long) k,
                    k < 0 ? "-" : "",
                    (unsigned long) (pa_bytes_to_usec(abs_k, ss) / PA_USEC_PER_MSEC),
                    pa_strnull(dn));
             pa_xfree(dn);
-            pa_alsa_dump(PA_LOG_ERROR, pcm);
+            pa_alsa_dump(PA_LOG_DEBUG, pcm);
         } PA_ONCE_END;
 
         /* Mhmm, let's try not to fail completely */
@@ -1253,13 +1253,13 @@
 
             PA_ONCE_BEGIN {
                 char *dn = pa_alsa_get_driver_name_by_pcm(pcm);
-                pa_log(_("snd_pcm_avail() returned a value that is exceptionally large: %lu bytes (%lu ms).\n"
+                pa_log_debug(_("snd_pcm_avail() returned a value that is exceptionally large: %lu bytes (%lu ms).\n"
                          "Most likely this is a bug in the ALSA driver '%s'. Please report this issue to the ALSA developers."),
                        (unsigned long) k,
                        (unsigned long) (pa_bytes_to_usec(k, ss) / PA_USEC_PER_MSEC),
                        pa_strnull(dn));
                 pa_xfree(dn);
-                pa_alsa_dump(PA_LOG_ERROR, pcm);
+                pa_alsa_dump(PA_LOG_DEBUG, pcm);
             } PA_ONCE_END;
 
             /* Mhmm, let's try not to fail completely */
@@ -1312,13 +1312,13 @@
                     k >= pa_bytes_per_second(ss)*10))
         PA_ONCE_BEGIN {
             char *dn = pa_alsa_get_driver_name_by_pcm(pcm);
-            pa_log(_("snd_pcm_mmap_begin() returned a value that is exceptionally large: %lu bytes (%lu ms).\n"
+            pa_log_debug(_("snd_pcm_mmap_begin() returned a value that is exceptionally large: %lu bytes (%lu ms).\n"
                      "Most likely this is a bug in the ALSA driver '%s'. Please report this issue to the ALSA developers."),
                    (unsigned long) k,
                    (unsigned long) (pa_bytes_to_usec(k, ss) / PA_USEC_PER_MSEC),
                    pa_strnull(dn));
             pa_xfree(dn);
-            pa_alsa_dump(PA_LOG_ERROR, pcm);
+            pa_alsa_dump(PA_LOG_DEBUG, pcm);
         } PA_ONCE_END;
 
     return r;
